packages/components/src/settings-form/types.ts

```diff
@@ -0,0 +1,32 @@
+export interface StringToString {
+	[ key: string ]: string;
```

Comment:
I think this typename is a bit misleading and you're better off relying on a built in or utility TS type than reusing this everywhere.

You could just use the Record type where you need it. such as `Record<string, string>`

Reply:
Thanks Sam, this felt a bit off to me as well 👍🏻